README.rst

```diff
@@ -82,6 +82,24 @@ On a Debian based system, the installation is as follows::
     make && make install
     cd ..
 
+Installing the PostgresSQL session_exec extension
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is optional, but required if you choose to install the python packages
+into a virtualenv.
+
+::
+
+    git clone https://github.com/okbob/session_exec
+    cd session_exec
+    make USE_PGXS=1 && make USE_PGXS=1 install
```

Comment:
You mean it works without `sudo`, so it's for the user to determine whether to use `sudo` or not?